src/ext/shared/dev.js

```diff
@@ -64,6 +66,7 @@ function generateFile(
 	};
 }
 
+/** @type {Array} */
```

Comment:
I assume you set this to an array of `any` to avoid errors further down in the file. 

 I understand the goal of this is to do _some_ type checking, but I would avoid `any` at all costs if possible since it nullifies the value type checking brings. In this case you have a few options but the suggestion below, while no perfect, I think is a better alternative to `any`. 

Again, this is not a big deal, but I just wanted to show you an alternative, it is your decision if it is worth it in this file or not.
```suggestion
/**
 * @typedef {Object} File
 * @property {string} content - Indicates whether the Courage component is present.
 * @property {string} filename - Indicates whether the Power component is present.
 * @property {number} lastModified - Indicates whether the Wisdom component is present.
 * @property {string} [name] - Indicates whether the Wisdom component is present.
 * @property {boolean} [request] - Indicates whether the Wisdom component is present.
 * @property {"css" | "js"} [type] - Indicates whether the Wisdom component is present.
 */
/** @type {File[]} */
```